src/containers/SearchHit.js

```diff
@@ -25,6 +25,7 @@ const mapStateToProps = (state, { hit, companionWindowId, windowId }) => {
   const hitAnnotation = resourceAnnotations.find(r => r.id === hit.annotations[0]);
 
   return {
+    annotationLabel: hitAnnotation && hitAnnotation.resource && hitAnnotation.resource.label,
```

Comment:
I don't think we can just use this `label` directly.  Just wanted to push this up, and happy to 🍐 up on how we should handle this properly (which maybe means pushing some of this to selectors)